tests/output/webpage/test_utils.js

```diff
@@ -1,29 +1,34 @@
-var runTest = function(options) {
-    if (options.version === undefined) {
-        options.version = ScratchpadConfig.prototype.latestVersion();
-    }
+import LiveEditorOutput from "../../../js/output/shared/output.js";
+import WebpageOutput from "../../../js/output/webpage/webpage-output.js";
+
+LiveEditorOutput.registerOutput("webpage", WebpageOutput);
 
-    var displayTitle = options.title +
-        " (Version: " + options.version + ")";
+export function createLiveEditorOutput(extraOptions) {
```

Comment:
FYI, you can use default args now, e.g.
```
function createLiveEditorOutput(extraOptions = {})
```

Reply:
Yay! JS and Python had a beautiful ES6 baby!

Reply:
I think we're okay in this case, but we'd definitely want to avoid this in python.